### ISSUE TITLE
Missing LICENSE prevents distribution

### ISSUE THREAD

**Issue by alita1991** [OT-CONTAINER-KIT/redis#89]
Hi,

I recently discovered that I could not distribute the product I'm working on, which uses Redis, because the project's LICENSE file is missing.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license

Can anyone please add a LICENSE file to the repository?

Thanks

**Comment by lsoica** [comment]
Good point! I wasn’t aware of this limitation regarding the missing LICENSE file. Was this an intentional decision for the repository, or might it have been overlooked?

**Comment by drivebyer** [comment]
Please take a look at this @shubham-cmyk @iamabhishek-dubey

**Comment by lsoica** [comment]
@shubham-cmyk @iamabhishek-dubey Could you please share your perspective? As far as I can tell, this prevents anyone from using the Redis operator with these images. Instead, it seems one would need to build their own Redis images. Is that correct?
